packages/data-point/lib/accumulator/factory.js

```diff
@@ -8,11 +8,15 @@ const LocalsFactory = require('../locals/factory')
  */
 function Accumulator () {
   this.value = undefined
+  this.resolvedValue = null
   this.locals = undefined
   this.values = undefined
   this.reducer = undefined
   this.trace = false
   this.context = undefined
+  this.resolveTransformWith = function (value) {
+    return { value: this.resolvedValue || value, isResolved: true }
```

Comment:
could you make this 2 lines please? 😄 got tripped up for a bit processing the double pipes and comma so close to each other